tests/countries/test_ivory_coast.py

```diff
@@ -0,0 +1,184 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+from unittest import TestCase
+
+from holidays.countries import IvoryCoast, CI, CIV
+from tests.common import CommonCountryTests
+
+
+class TestIvoryCoast(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(IvoryCoast, years=range(1997, 2050))
+
+    def test_country_aliases(self):
+        self.assertAliases(IvoryCoast, CI, CIV)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(IvoryCoast(years=1959))
+
+    def test_2025(self):
+        self.assertHolidays(
+            IvoryCoast(years=2025),
+            ("2025-01-01", "1er janvier"),
+            ("2025-03-27", "Lendemain de la Nuit du Destin"),
+            ("2025-03-30", "Fête de fin du Ramadan"),
+            ("2025-03-31", "Lendemain de la Fête de fin du Ramadan"),
+            ("2025-04-21", "Lundi de Pâques"),
+            ("2025-05-01", "Fête du travail"),
+            ("2025-05-29", "Jour de l’Ascension"),
+            ("2025-06-06", "Fête de la Tabaski"),
+            ("2025-06-09", "Lundi de Pentecôte"),
+            ("2025-08-07", "Fête Nationale"),
+            ("2025-08-15", "Fête de l’Assomption"),
+            (
+                "2025-09-04",
+                "Lendemain de l’Anniversaire de la Naissance du Prophète Mahomet",
+            ),
+            ("2025-11-01", "Fête de la Toussaint"),
+            ("2025-11-15", "Journée Nationale de la Paix"),
+            ("2025-12-25", "Fête de Noël"),
+        )
+
+    def test_new_years_day(self):
+        self.assertHolidayName("1er janvier", (f"{year}-01-01" for year in range(1997, 2050)))
+
+    def test_labor_day(self):
+        self.assertHolidayName("Fête du travail", (f"{year}-05-01" for year in range(1997, 2050)))
+        self.assertHolidayName("Lendemain de la Fête du travail", "2022-05-02")
```

Comment:
```suggestion
        name = "Fête du travail"
        self.assertHolidayName(name, (f"{year}-05-01" for year in range(1997, 2050)))
        dt = (
            "2005-05-02",
            "2011-05-02",
            "2016-05-02",
            "2022-05-02",
        )
        self.assertHolidayName(f"Lendemain de la {name}", dt)
        self.assertNonObservedHoliday(dt)
```
And the same for similar holidays.